bundle/edu.gemini.p1backend/TODO.md

```diff
@@ -0,0 +1,50 @@
+# Short term
+
+Something that improves the situation now. Nothing for the future. A dead-end by
+design. Something to keep chugging away until we do it right. It shouldn't
+take more work than 2 weeks.
+
+1. Code lives in ocs.
+1. URLs:
+   - Receiver of P1 submissions: phase1.gemini.edu/receive/.../...
```

Comment:
note that this url isn't user facing but still it is a good idea to make them restish